src/editor/components/modals/file-modal/manage-file-sources/__stories__/manage-file-sources.stories.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
```

Comment:
I realized only a couple of hours ago that we need to be using Storybooks for _all_ component development, including this one. We don't have an accepted way of organizing these storybooks, so read through this file with that caveat in mind. I was not comprehensive by _any_ means with this one – merely playing with the idea and adding the components that I needed to really decouple from the app for further development to get this more cleaned up. A more thorough PR would incorporate the other parts of the `ManageFileSources` component as well as the component _in toto_. I'm happy to do that down the line but probably not for this PR.